### ISSUE TITLE
[Feature Request] Delete multiple databases

### ISSUE THREAD

**Issue by obrunsmann** [TablePlus/TablePlus#2651]
Hello :)

While working locally, I regularly have the problem that I want to delete several databases at the same time. However, in the database window (CMD+K) I cannot make a multiselect to delete multiple databases.

Regards

**Comment by jtomek** [comment]
Hello, this feature would be very nice to have. 

<img width="424" alt="image" src="https://user-images.githubusercontent.com/3748234/194135256-bd206951-8678-4a32-b99c-e162fc9224aa.png">


The workaround is to run the following sequence:

1: 

```mysql
SHOW DATABASES WHERE `Database` LIKE "LOCAL_%"
```

2: Copy and paste the result, insert it to VSCode, prepend with `drop database`, add a `;` at the end of each line.

```mysql
...
drop database LOCAL_351596233166422016;
drop database LOCAL_352229449590112256;
drop database LOCAL_362858268747366400;
drop database LOCAL_362858371767861248;
drop database LOCAL_378303064605458432;
drop database LOCAL_382429401159041024;
drop database LOCAL_382776074167123968;
...
```

3. paste to TablePlus, select all lines and execute them

This process is rather tiresome.

**Comment by jtomek** [comment]
Hello @huyphams, thank you for the reply. I am sorry but I don't understand what you mean that TablePlus "supports multiple carets". Could you please explain what it means?

**Comment by huyphams** [comment]
I thought you paste it to VSCode to edit multiple lines? We can do it within TablePlus, see this blog: https://tableplus.com/blog/2019/10/whats-new-in-build-270.html

**Comment by jtomek** [comment]
Yes, I pasted it to VSCode to edit multiple lines. But that's redundant now, thank you! I didn't know about these shortcuts. Nice!

Nonetheless, the possibility of deleting multiple databases (maybe only if the tag is local) from the UI would be handy too. :-) But I understand this feature is probably not as straightforward as one would think.

**Comment by MPGs007** [comment]
how do you multi select in windows - ctrl shift not working like on mac ?

**Comment by MPGs007** [comment]
select all drop - confirm would be very handy

**Comment by ilieandriuta** [comment]
This would be so helpful! I started using TablePlus not long ago and I miss this feature I used to have and which is very handy for me because I use a lot of databases.. I honestly didn't know you can't delete multiple databases at once on TablePlus....
